src/natcap/invest/sdr/sdr_core.pyx

```diff
@@ -419,49 +455,59 @@ def calculate_sediment_deposition(
         xoff = offset_dict['xoff']
         yoff = offset_dict['yoff']
 
-        LOGGER.info('Sediment deposition %.2f%% complete', 100.0 * (
+        LOGGER.info('Sediment deposition %.2f%% complete', 100 * (
             (xoff * yoff) / float(n_cols*n_rows)))
 
         for row_index in range(win_ysize):
             seed_row = yoff + row_index
             for col_index in range(win_xsize):
                 seed_col = xoff + col_index
-                # search to see if this is a good seed
-                if mfd_flow_direction_raster.get(seed_col, seed_row) == 0:
+                # check if this is a good seed
+                # a good seed pixel is a local high point
+                if mfd_flow_direction_raster.get(seed_col, seed_row) == mfd_nodata:
```

Comment:
Does a local high point, a good seed pixel, just mean that it is not `nodata` and drains downstream?

Reply:
rephrased this comment slightly